src/persist/src/operators/stream.rs

```diff
@@ -287,68 +269,44 @@ where
                     return;
                 }
 
-                // Only try and seal if we have seen some data. Otherwise, we wouldn't have
-                // a capability that allows us to emit errors.
-                if let Some(err_cap) = capabilities.get(0) {
-                    for frontier_element in new_input_frontier.iter() {
-                        // Only seal if this element of the new input frontier truly
-                        // represents progress. With Antichain<u64>, this will always be
-                        // the case, but antichains of types with a different partial order
-                        // can have frontier progress and have some elements that don't
-                        // represent progress.
-                        if !input_frontier.less_than(frontier_element) {
-                            continue;
-                        }
+                for frontier_element in new_input_frontier.iter() {
+                    // Only seal if this element of the new input frontier truly
+                    // represents progress. With Antichain<u64>, this will always be
+                    // the case, but antichains of types with a different partial order
+                    // can have frontier progress and have some elements that don't
+                    // represent progress.
+                    if !input_frontier.less_than(frontier_element) {
+                        continue;
+                    }
 
-                        log::trace!("Sealing {} up to {}", &operator_name, frontier_element);
+                    log::trace!("Sealing {} up to {}", &operator_name, frontier_element);
 
-                        let result = write.seal(*frontier_element).await;
-                        if let Err(e) = result {
-                            log::error!(
-                                "Error sealing {} up to {}: {:?}",
-                                &operator_name,
-                                frontier_element,
-                                e
-                            );
+                    if let Err(e) = write.seal(*frontier_element).await {
+                        log::error!(
+                            "Error sealing {} up to {}: {:?}",
+                            &operator_name,
+                            frontier_element,
+                            e
+                        );
 
-                            let mut session = error_output.session(err_cap);
-                            // TODO: Make error retractable? Probably not...
-                            session.give((e.to_string(), *err_cap.time(), 1));
-                        }
+                        return;
```

Comment:
add a comment explaining that if this fails, we will simply retry the next time the operator is invoked?